src/BuildScriptGenerator/Python/PythonBashBuildSnippet.sh.tpl

```diff
@@ -1,86 +1,96 @@
 declare -r TS_FMT='[%T%z] '
 declare -r REQS_NOT_FOUND_MSG='Could not find requirements.txt; Not running pip install'
 echo "Python Version: $python"
-cd "$SOURCE_DIR"
 
-{{ if VirtualEnvironmentName | IsNotBlank }}
+virtualEnvName="{{ VirtualEnvironmentName }}"
```

Comment:
@gpcastro , @dorfire , @arroyc : What do you guys think of this approach? Earlier we used to hide the parts of the script from being generated, but here we print the whole script but either enable/disable the parts using these flags. My thought was that this way if a user wants to run the generated script, at least he would know what the whole script was and can see which logic runs when.

Reply:
After talking with @gpcastro I reverted to what we had earlier before